### ISSUE TITLE
Project overview missing folders in generated project documentation (0.11.0rc2)

### ISSUE THREAD

**Issue by swiffer** [fishtown-analytics/dbt#976]
### Issue description
The Tutorial section on getdbt.com is showing a tree navigation for the new auto generated documentation website. this is not present currently - not yet implemented or something not set up correctly?
https://docs.getdbt.com/v0.11/docs/documentation-website

### Results
![grafik](https://user-images.githubusercontent.com/2990373/45094865-e42aa200-b11c-11e8-870f-7a8676d7fb41.png)

![grafik](https://user-images.githubusercontent.com/2990373/45094874-ed1b7380-b11c-11e8-8106-8c33fb5297c5.png)

### System information
The output of `dbt --version`:
```
installed version: 0.11.0-rc2
   latest version: 0.10.2

Your version of dbt is ahead of the latest release!
```

**Comment by drewbanin** [comment]
Hmm, this is pretty weird. I would definitely expect the `base/` and `tables/` directories to show up in the docs site.

What kind of operating system are you using?

And can you share the contents contents of your `dbt_project.yml` file? I'm mainly interested in the `source-paths` config option.

**Comment by swiffer** [comment]
using win10 64bit with powershell

dbt_project.yml
```yaml
source-paths: ["models"]
```

**Comment by drewbanin** [comment]
Ok - I think this is because Windows uses different path separators than Linux.

I think I have a fix for this, but I don't have a Windows machine handy. Are you able to pop open the `target/manifest.json` file and send over an example `original_file_path` entry? So if your manifest.json looks like:

```
{
    "nodes": {
        "model.test.test_model": {
            "description": "",
            "resource_type": "model",
            "package_name": "test",
            "original_file_path": "models/test_model.sql",
            "tags": [],
            ...
      }
...
```

I'd be interested in the `original_file_path` entry. My guess is that it would look like `"models\\test_model.sql"`, but an example would make me a lot more confident!

**Comment by swiffer** [comment]
seems to be fixed in 0.11.0 already?

**Comment by drewbanin** [comment]
thanks @swiffer! Yeah - we snuck in a quick fix for this yesterday. Going to close this issue -- thanks for brining it to our attention!
